<launch>
  <arg name="scene_name" default="2n8kARJN3HM"/>
  <arg name="path_to_gt_map" default="/home/kirill/TopoSLAM/GT/$(arg scene_name)"/>
  <arg name="path_to_load_json" default="/home/kirill/TopoSLAM/toposlam_ws/data/habitat_mipt_bags/mp3d_rlnav/$(arg scene_name)"/>
  <arg name="path_to_save_logs" default="/home/kirill/TopoSLAM/toposlam_ws/data/prism_topomap_logs"/>
  <arg name="config_file" default="habitat_mp3d_localization.yaml"/>

  <node pkg="prism_topomap" type="prism_topomap_node.py" name="prism_topomap_node" output="screen">
    <param name="path_to_gt_map" value="$(arg path_to_gt_map)"/>
    <param name="path_to_load_graph" value="$(arg path_to_load_json)"/>
    <param name="path_to_save_logs" value="$(arg path_to_save_logs)"/>
    <param name="config_file" value="$(arg config_file)"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="tf_map_to_odom" args="0 0 0 0 0 0 map odom 100"/>
</launch>